### PR TITLE
Adding PIP support for PythonUWP projects for VS

### PR DESCRIPTION
**Pull request by alanch-ms**
- Added a win32 python interpreter into the build for package download and intellisense
- Add target, CreatePythonUwpIoTPythonEnv, to copy the python interpreter into the project folder

### CODE REVIEW

**Comment by zooba** [comment]
LGTM

**Comment by alanch-ms** [comment]
@munyirik , @alecont , CR please.

**Comment by munyirik** [comment]
Sorry if you mentioned before but why does the interpreter need to be packaged in the project folder?

**Comment by alanch-ms** [comment]
The (win32) interpreter is needed so that the user can download PIP packages on the PC.  The interpreter is also used for intellisense.

**Comment by munyirik** [comment]
My question was about the packaging vs. using a python installation.

**Comment by munyirik** [comment]
👍

File: cpython/Tools/pyuwp/pyuwpsdk/pyuwpsdk.csproj [hunk 1]
```diff
@@ -23,6 +23,7 @@
     <CopyBuildOutputToOutputDirectory>false</CopyBuildOutputToOutputDirectory>
     <CopyOutputSymbolsToOutputDirectory>false</CopyOutputSymbolsToOutputDirectory>
     <InputDir>$(SolutionDir)uwp\10.0</InputDir>
+    <InterpreterDir Condition="'$(InterpreterDir)' == ''">..\..\..\PCBuild\win32</InterpreterDir>
```
Comment (alecont) [inline cpython/Tools/pyuwp/pyuwpsdk/pyuwpsdk.csproj:26]: is this inside a "win32" condition? I assume so...

**Comment by alecont** [comment]
minor comment, for the rest looks good :-)

File: cpython/Tools/pyuwp/pyuwpsdk/pyuwpsdk.csproj (hunk 1 above)
Comment (alanch-ms) [inline cpython/Tools/pyuwp/pyuwpsdk/pyuwpsdk.csproj:26]: This is the pyuwpsdk project --  the project that builds the VSIX.  Our python UWP VSIX contains 6 PythonUWP configurations ( [x86|x64|arm] x [Debug|Release]) and 1 win32/x86 python.  The win32 python which is used for intellisense and package downloading only.  The "InterpreterDir" is used for the one and only win32 python interpreter.  Hence, it doesn't need to be under "win32" condition.

Comment (alecont) [inline cpython/Tools/pyuwp/pyuwpsdk/pyuwpsdk.csproj:26]: ok good

**Comment by alecont** [comment]
LGTM

**Pull request merged**